lib/controllers/frontend/spree/checkout_controller_decorator.rb

```diff
@@ -1,40 +1,43 @@
-require 'spree/core/validators/email'
-Spree::CheckoutController.class_eval do
-  before_filter :check_authorization
-  before_filter :check_registration, :except => [:registration, :update_registration]
+if defined? Spree::Frontend
```

Comment:
If I remember correctly checking `defined? Spree::Frontend` caused me issues when I did the initial work splitting this frontend & backend apart.  Might be safer to use:
`if Spree::Auth::Engine.frontend_available?`

https://github.com/spree/spree_auth_devise/blob/master/lib/spree/auth/engine.rb#L60-L62